### DISCUSSION TITLE
Annotation updating

### DISCUSSION THREAD

**Discussion by simpson-david** [six7/figma-tokens#545] [Ideas]
Hi, I love the annotation feature so thanks for that. A cool upgrade for it would be for the annotation to be linked to the component so that if the component updates so do the token references listed. the annotation currently needs deleting and regenerating if a change is made.
A. an I right about this?
B. is it on a roadmap?

Thanks

**Comment by six7** [comment]
Sorry for missing this. It's on the roadmap and is coming up soon! 🎉

**Reply by simpson-david** [reply]
Awesome, thanks :)
